jetstream/core/config_lib.py

```diff
@@ -52,6 +53,11 @@ class InstantiatedEngines:
   interleaved_engines: List[engine_api.Engine]
 
 
+@dataclasses.dataclass
+class MetricsServerConfig:
```

Comment:
So now we pass in MetricsServerConfig to enable prometheus metrics, could you add a README in `/docs` to describe how to enable and use the prometheus metrics for future users? It would be very helpful for our customers.